src/analysis/subspaces/determineBinaryMatrix.m

```diff
@@ -0,0 +1,35 @@
+function [Shat, Shatabs, mconnect, nconnect, mconnectin, mconnectout] = determineBinaryMatrix(S)
+% Determine the binary form of the stoichiometric matrix S
+%
+% INPUT:
+%
+%    S:            Stoichiometric matrix of size m x n (m rows, n columns)
+%
+% OUTPUTS:
+%
+%    Shat:         Binary form of the stoichiometric matrix
+%    Shatabs:      Absolute value of Shat
```

Comment:
From the name of the function, I would actually expect it to only return Shatabs, or even just a logical sparse with `Sbin = S~=0`

Reply:
What other name do you suggest?

Reply:
`determineSignMatrix`?

Reply:
Ok I can change that.